pkg/cluster_upgrader/cluster_upgrader.go

```diff
@@ -320,21 +320,25 @@ func CreateWorkerMaintWindow(c client.Client, m maintenance.Maintenance, upgrade
 }
 
 // This check whether all the master nodes are ready with new config
-func AllMastersUpgraded(c client.Client, m maintenance.Maintenance, upgradeConfig *upgradev1alpha1.UpgradeConfig, reqLogger logr.Logger) (bool, error) {
+func AllMastersUpgraded(c client.Client, ms metrics.Metrics, m maintenance.Maintenance, upgradeConfig *upgradev1alpha1.UpgradeConfig, reqLogger logr.Logger) (bool, error) {
 
 	return NodesUpgraded(c, "master", upgradeConfig, reqLogger)
 
 }
 
 // This check whether all the worker nodes are ready with new config
-func AllWorkersUpgraded(c client.Client, m maintenance.Maintenance, upgradeConfig *upgradev1alpha1.UpgradeConfig, reqLogger logr.Logger) (bool, error) {
-
-	return NodesUpgraded(c, "worker", upgradeConfig, reqLogger)
+func AllWorkersUpgraded(c client.Client, ms metrics.Metrics, m maintenance.Maintenance, upgradeConfig *upgradev1alpha1.UpgradeConfig, reqLogger logr.Logger) (bool, error) {
+	ok, err := NodesUpgraded(c, "worker", upgradeConfig, reqLogger)
```

Comment:
Small nitpick. Can you lowercase `NodesUpgraded` -> `nodesUpgraded` and I think it no longer requires the upgrade param